paper/template.tex

```diff
@@ -82,17 +86,40 @@
 % NOTE: have to careful when doing time comparisons as currently I am doing it as time windows so "last 2 years" doesn't work as that is 2 years of data not 1 year
 
 \begin{abstract}
-  Continuos integration (CI) is becoming more popular as software development moves to an Agile fast paced development life cycle. Most CI is done automatically using a service which runs based off configuration. Our major questions is how much is CI actually being used? As well as how are these files being structured? We got 94,379 open source projects from Github to answer these questions. In doing so compared our results against \citet{Hilton2016} work to see if their has been a increase in usage. We found a shift in CI services being used and were able to get similar results to their study. In terms of structure we found that configuration files are written with no comments normally. We suggest at the end further research is needed to get a better understanding of this growing field.    
+  \sm{is Sanner double blind?}
+
+Continuous integration (CI) is widely popular
+for instance as support for fast-paced development cycle.
+\sm{do we want to frame the paper around a reproduction of the earlier study?
+Since this seems to be the strongest part, I would start out with this.
+Pose the question of whether things have changed since people looked at it last.}
+This paper investigates how CI is currently used by projects on GitHub.
+To our knowledge, the last major study was in 2016
+by \citeauthor{Hilton2016}\citep{Hilton2016}
+and we want to see how things changed in the last 5 years.
+% In doing so compared our results against \citet{Hilton2016} work to see if their has been a increase in usage.
+
+We analyze 94,379 open source projects from GitHub % to answer these questions.
+to understand how CI is used based on the corresponding configuration files.
+
+% Most CI is done automatically using a service which runs based off configuration.
+% Our major questions is how much is CI actually being used?
+%As well as how are these files being structured?   
+We found a shift in CI services being used.\sm{can we give an example here?
+trying to summarize the results a little}
```

Comment:
Yep, the shift is more projects are using GithubActions as opposed to CircleCI which was previously the 2nd most used service. Travis CI is still the most commonly used service in our sample.